src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -105,6 +106,61 @@
             });
         }
 
+        #createTableSearchFields() {
+            document.querySelectorAll('div.table-with-search-field').forEach((table, i) => {
+                const searchField = document.createElement('input');
+                searchField.type = 'search';
+                searchField.placeholder = 'search...';
+                searchField.id = `table-search-field-${i}`;
+                searchField.classList.add(`table-search-field-input`);
+                searchField.autocapitalize = 'off';
+                searchField.autocomplete = 'off';
+                searchField.autocorrect = 'off';
+                table.insertBefore(searchField, table.firstChild);
+
+                const labelField = document.createElement('label');
+                labelField.htmlFor = `table-search-field-${i}`;
+                labelField.classList.add(`table-search-field-label`);
+                labelField.innerHTML = 'Search inside the contents of the table';
+                table.insertBefore(labelField, table.firstChild);
+
+                searchField.addEventListener('input', () => {
```

Comment:
I suggest to add a small debounce... (see https://kettanaito.com/blog/debounce-vs-throttle if you want a really great interactive blog post on the subject)

Reply:
I've read the article. Thanks for sharing.

However, for this use case, I think he added complexity is not worth it, so I'd prefer to not implement that for now. Thanks.